packages/webapp/nginx/sites/default.conf

```diff
@@ -5,4 +5,8 @@ server {
         root /usr/share/nginx/html;
         try_files $uri /index.html;
     }
+
+    location /api {
```

Comment:
This redirect already added in `docker/nginx/sites/server.template`

Reply:
Yeah this was due to not wanting an extra container just for nginx. Depending on the setup there are a few interesting ways to run ingress.